packages/vscode-extension/src/commonlib/azureLogin.ts

```diff
@@ -302,6 +309,18 @@ export class AzureAccountManager extends login implements AzureAccountProvider {
         if (item.subscription.subscriptionId == subscriptionId) {
           AzureAccountManager.tenantId = item.session.tenantId;
           AzureAccountManager.subscriptionId = subscriptionId;
+          TreeViewManagerInstance.getTreeView("teamsfx-accounts")!.refresh([
```

Comment:
Curious about the further plan of UI update - will the `statusMap` be retired after the componentize change?
If yes, ok to directly put UI operation in `azureLogin`.
Otherwise, keep `statusMap` and move UI operation out of `azureLogin`.

Reply:
For now,  ```statusMap``` only notify login and logout message. TreeView subscription information can only be updated by the direct call command. I think later we can add more information to ```notify logic```.